api/grpcserver/smesher_service.go

```diff
@@ -74,15 +74,15 @@ func (s *SmesherService) SetPostServiceConfig(cfg activation.PostSupervisorConfi
 }
 
 // IsSmeshing reports whether the node is smeshing.
-func (s SmesherService) IsSmeshing(context.Context, *emptypb.Empty) (*pb.IsSmeshingResponse, error) {
+func (s *SmesherService) IsSmeshing(context.Context, *emptypb.Empty) (*pb.IsSmeshingResponse, error) {
```

Comment:
Most other services like `NodeService`, `GlobalStateService` and `MeshService` all also take value receivers and might benefit from the change to pointer receivers.